lib/index.js

```diff
@@ -30,12 +30,8 @@ var defaults = {
       'ios > 6, android > 4.3, samsung > 3, chromeandroid > 50'
   },
   'postcss-easy-import': {
-    onImport: function(imported) {
-      // Update the watch task with the list of imported files
-      if (typeof global.watchCSS === 'function') {
-        global.watchCSS(imported);
-      }
-    }
+    transform: identity,
```

Comment:
We definitely need it to add `debug` but in this case is unnecessary extra work – probably the overhead is ~0 so not a big deal I just figured I'd ask